homeassistant/components/asuswrt/strings.json

```diff
@@ -88,6 +88,48 @@
       },
       "6ghz_temperature": {
         "name": "6GHz Temperature"
+      },
+      "cpu_usage": {
+        "name": "Cpu usage"
```

Comment:
Since CPU is an acronym for central processing unit we should upper case it.

Reply:
Changed as requested